index.html

```diff
@@ -86,9 +89,11 @@ <h4>Preferences</h4>
                 <div class="projects col s12 m6"></div>
                 <div class="students col s12 m6"></div>
               </div>
-
             </div>
           </div>
+          <footer class="link-container">
+            <a href="http://www.freepik.com" class="graphic">Graphic designed by Starline / Freepik</a>
```

Comment:
Change the attribution to:
`<a href="https://www.freepik.com/free-photos-vectors/background">Background vector created by Starline - Freepik.com</a>`
To avoid any copyrights breaches

Reply:
no problem! I will update it.